vmm/src/device_manager.rs

```diff
@@ -4239,16 +4239,6 @@ impl Aml for DeviceManager {
             mbrd_memory_refs.push(mbrd_memory_ref as &dyn Aml);
         }
```

Comment:
You can also remove all the code above this (populating `mbrd_memory` and `mbrd_memory_refs`). Not sure why the compiler/clippy didn't notice they were unused.

Reply:
I've updated the patch by not removing the MBRD.

Reply:
Why did you drop the removal? I was pointing out that the code that was used to generate the inner content of the device could also be removed (I don't know why the compiler didn't notice.)

Reply:
I see, the last paragraph in your text.